lib/ably/realtime/channel.rb

```diff
@@ -179,15 +184,26 @@ def presence
         @presence
       end
 
-      # Return the message history of the channel.
+      # Return the message history of the channel
+      #
+      # Once attached to a channel, you can retrieve messages published on the channel before the
```

Comment:
I am not sure I agree on your second comment.  Imagine a method with an argument definition such as:
```ruby
# id is required 
def get_thing(id)
end
```

Do we really need to tell people what will happen if they don't follow the rules of a method definition?